extensions/amp-video/0.1/flexible-bitrate.js

```diff
@@ -35,6 +35,9 @@ const BITRATE_BY_EFFECTIVE_TYPE = {
   '5g': 5000,
 };
 
+/** @const {number} The percentage that videos need to be buffered to not downgrade them */
```

Comment:
I think this comment could be a bit easier to understand with the important information at the beginning: "Do not downgrade the quality of a video that has loaded enough content"